pkg/trace/agent/agent.go

```diff
@@ -605,6 +612,48 @@ func isManualUserDrop(pt *traceutil.ProcessedTrace) bool {
 	return dm == manualSampling
 }
 
+// errorSampling reports trace chunks with errors and tags spans as Error Tracking Backend Standalone.
+// Also sets "DroppedTrace" on the chunk.
+func (a *Agent) errorSampling(now time.Time, ts *info.TagStats, pt *traceutil.ProcessedTrace) (keep bool, numEvents int) {
+	sampled := a.runErrorSampler(now, *pt)
+	numEvents = len(a.getAnalyzedEvents(pt, ts))
+	if sampled {
+		for _, span := range pt.TraceChunk.Spans {
+			if span.Error != 0 || spanContainsExceptionSpanEvent(span) {
+				span.Meta["_dd.error_tracking_backend_standalone.error"] = "true"
+			} else {
+				span.Meta["_dd.error_tracking_backend_standalone.error"] = "false"
```

Comment:
We are going to need the tag for billing, there is no other way we could know they come from ETS (we don't want to bill them for APM)

Reply:
You can have a single tag or flag at the `TraceChunk.Tags` level does represents the sampling mechanism (Standalone Error Sampling), and then, on the Intake, resolve whether the error/exception is found on any given span, then propagate the information as needed. Wouldn't this make the transport more efficient for both DD and customers?

Reply:
Sounds good, tagged the chunk instead of each span in https://github.com/DataDog/datadog-agent/pull/30065/commits/96d9668e722c806ed325dc2ae8f6950e363eae39